README.md

```diff
@@ -0,0 +1 @@
+Hello world
```

Comment:
- [ ] Please add descriptive Readme. This is one of the basics requirements here at Microverse. Please go back and have look at the project requirements.